reminders/class_tests/create_reminder_tests.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from reminders.forms import ReminderCreationForm
+from pytest_django.asserts import assertTemplateUsed
+
+
+@pytest.mark.django_db
+class TestCreateReminder:
+
+    @pytest.mark.parametrize("invalid_user_credentials", [
+        # method cannot be None
+        {'participant_id': None, 'date_time': pytest.valid_date_time,
+         'messages': pytest.message, 'method': None
+         },
+        # method can be only from ReminderType
+        {'participant_id': None, 'date_time': pytest.valid_date_time,
+         'messages': pytest.message, 'method': pytest.invalid_method
+         },
+        # datetime should be greater then the present time
+        {'participant_id': None, 'date_time': pytest.invalid_date_time,
+         'messages': pytest.message, 'method': pytest.valid_method
+         }
+    ], ids=[
+        "method is none",
+        "method is not from ReminderType",
+        "date is before the current time"
+        ]
+    )
+    def test_create_reminder(self, get_event_participant, invalid_user_credentials):
```

Comment:
please change the method name to creating invalid reminder, also if possible make also create a test for a valid form

Reply:
done, added a new method, and change the old one name.